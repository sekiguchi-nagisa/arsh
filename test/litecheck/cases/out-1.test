
# RUN: /bin/sh $self
echo hello
# CHECK: hell