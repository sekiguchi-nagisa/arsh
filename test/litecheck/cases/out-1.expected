CHECK on line 4,
@@ -1 +1 @@
-hell
+hello
stdout is mismatched